build.gradle

```diff
@@ -3,6 +3,9 @@ buildscript {
         mavenLocal()
         google()
         mavenCentral()
+        maven {
+            url "https://plugins.gradle.org/m2/"
```

Comment:
I think `gradlePluginPortal()` resolves to this?

Reply:
Yeah, I think you're right. I might have looked at some old documentation. I'll try changing this.

Reply:
Changed